src/api/che-login-manager.ts

```diff
@@ -1,12 +1,14 @@
-/*********************************************************************
- * Copyright (c) 2020 Red Hat, Inc.
- *
+/**
+ * Copyright (c) 2019-2021 Red Hat, Inc.
```

Comment:
Shouldn't it be `2020-2021`?

Reply:
Old files should be 2019-2021
New files 2021-2021?

Reply:
This file was changed automatically by eslint